jujugui/static/gui/src/app/app.js

```diff
@@ -2067,8 +2067,8 @@ YUI.add('juju-gui', function(Y) {
      *
      */
     authorizeCookieUse: function(req, res, next) {
-      var ga_key = this.get('GA_key');
-      if (ga_key) {
+      var GTM_enabled = this.get('GTM_enabled');
```

Comment:
Anything that's set in `juju_config` will get applied to app.js so this should work.
https://github.com/juju/juju-gui/blob/develop/jujugui/templates/index.html.mako#L398